source/Planet.cpp

```diff
@@ -120,7 +121,17 @@ void Planet::Load(const DataNode &node, Set<Wormhole> &wormholes, const Conditio
 			else if(key == "outfitter")
 				outfitSales.clear();
 			else if(key == "government")
-				government = nullptr;
+			{
+				bool keepTribute = false;
+				for(const DataNode &grand : child)
+				{
+					if(grand.Token(0) == "keep tribute")
```

Comment:
Can we just leave out this keyword? Basically just having the player always lose tribute whenever there is a government change?
Tribute is (currently) some sort of extra game, and I think it's fine to just always have the player lose tribute on government changes.

Reply:
I can think of an edge case where this might be useful, when you're assigning a government that is internally separate, but from the player's perspective is the same as the old one, and you don't want the player to know about that.

Reply:
Yes, but changes in planet governments are rare, and I have not seen any locations in-game where the planetary government changes but still stays the same. (Although maybe that was just the intention of authors.)

If we want this "change the government, but keep everything else the same", then we should use a more generic keyword, because there might be more in the future that we should hide. Maybe something like `"silent revolution"`, `"in name only"` or `"hidden revolt"`.

I think we also have automatic conditions for tribute. If those allow writing of zero, then we already have a way of relinquishing tribute in events (but that then needs to be done manually).

Reply:
Just checked it, we can set tribute through conditions. How about just changing the events where we want to relinquish tribute?